docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst

```diff
@@ -155,7 +157,10 @@ Scale-up of stomach cancer screening using ABC method with endoscopic follow-up
 
 :underline:`Alternative scenario`
 
-In the alternative scenario, there will be a scale up of ABC screening starting from 5% to 30% as indicated in the coverage figure below. We do not assume that the 5% who recieve H. pylori screening in the baseline is the same groups those who recieve screening in the alternative scenario. 
+In the alternative scenario, there will be a scale up of ABC screening starting from 5% to 30% as indicated in the coverage figure (orange line) below. 
+
+.. note::
+  We do not assume that the 5% who recieve H. pylori screening in the baseline is the same groups those who will recieve screening in the alternative scenario. (Hence H. pylori prevalence does not need to be split out between baseline screened or not screened).
 
```

Comment:
Oh, I missed this.  Does it mean we need to do some sort of "treatment-deleted incidence rate" for the alternative scenario?

Reply:
I don't think so. There is no evidence available to say that those who had H. pylori in the past (baseline) is more likely to partake in stomach cancer screening when available or vice versa (in the alternative scenario). For simplicity, I think we assume that the population who partake in screening is the same as those who do not in relation to their past H. pylori screening status.